page-objects/pages/coursePages/CRSSCourseListPage.js

```diff
@@ -272,15 +272,15 @@ async function studentLogsInAndGoesToTasksOfTheCourse(username, password, course
 	await goToTasksOfTheCourse(coursename);
 }
 
-async function verifyCourseAndTopic(coursename, topicname, section) {
+async function isTopicInCourse(coursename, topicname, section) {
 	await clickOnCourseInSection(coursename, section);
 	let topicNames = await Promise.all(
 		(await driver.$$('#topic-list > div > div > div')).map(async (element) => await element.getText())
 	);
 	await expect(topicNames).to.include(topicname);
 }
 
-async function verifyCopyWithStudents(coursename) {
+async function areStudentsInCopiedCourse(coursename) {
```

Comment:
The method checks that there are NO students in copied course

Reply:
done